packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx

```diff
@@ -42,6 +53,31 @@ export interface ChatMessageActionFlyoutProps {
  */
 export const ChatMessageActionFlyout = (props: ChatMessageActionFlyoutProps): JSX.Element => {
   const theme = useTheme();
+  const messageReadByList: IContextualMenuItem[] = [];
+
+  props.messageReadByNames?.forEach((person) => {
```

Comment:
nit: This is a good place to use a `map` function as we are converting one-for-one each person into a contextmenuitem:
```ts
const messageReadByList: IContextualMenuItem[] | undefined = props.messageReadByNames?.map((person) => {
   ...
   return {
     key: person.name,
     ...
   };
```

Reply:
changed code to use a map function